import-service/serverless.ts

```diff
@@ -0,0 +1,55 @@
+import type { AWS } from '@serverless/typescript';
+
+import importProductsFile from '@functions/import-products-file';
+import importFileParser from '@functions/import-file-parser';
+
+const serverlessConfiguration: AWS = {
+    service: 'import-service',
+    frameworkVersion: '2',
+    custom: {
+        webpack: {
+            webpackConfig: './webpack.config.js',
+            includeModules: true,
+        }
+    },
+    plugins: ['serverless-webpack'],
+    provider: {
+        name: 'aws',
+        runtime: 'nodejs14.x',
+        region: "eu-west-1",
+        apiGateway: {
+            minimumCompressionSize: 1024,
+            shouldStartNameWithService: true,
+        },
+        environment: {
+            AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+            S3_BUCKET: 'product-service-dev-csv',
+        },
+        lambdaHashingVersion: '20201221',
+        iamRoleStatements: [
+            {
+                Effect: 'Allow',
+                Action: 's3:ListBucket',
+                Resource: 'arn:aws:s3:::${self:provider.environment.S3_BUCKET}',
+            },
+            {
+                Effect: 'Allow',
+                Action: 's3:GetObject',
+                Resource: 'arn:aws:s3:::${self:provider.environment.S3_BUCKET}/*',
+            },
+            {
+                Effect: 'Allow',
+                Action: 's3:PutObject',
+                Resource: 'arn:aws:s3:::${self:provider.environment.S3_BUCKET}/*',
+            },
+            {
+                Effect: 'Allow',
+                Action: 's3:DeleteObject',
```

Comment:
Why not just `s3:*` instead of allowing get/put/delete actions separately?

Reply:
I always prefer to grant permissions the resource really needs instead of granting the said resource full access :) In case this labmda grows and requires more permissions I will probably grant full permissions, but In our case it's really not necessary :)